src/include/devcomm.h

```diff
@@ -177,6 +185,7 @@ struct ncclChannel {
       struct ncclRing ring;
       struct ncclTree tree;
       struct ncclTree collTree;
+      struct scklGraph graph;
```

Comment:
change graph to sgraph (as mentioned above).

Reply:
fixed.